charts/drone-runner-docker/values.yaml

```diff
@@ -0,0 +1,141 @@
+# Default values for drone-runner-docker.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+## The official drone docker runner image, change tag to use a different version.
+## ref: https://hub.docker.com/r/drone/drone-runner-docker/tags/
+##
+image:
+  repository: drone/drone-runner-docker
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 1.8.1
```

Comment:
wondering if we could set `tag: 1` so that we don't have to update the chart for every drone release.  `drone/drone-runner-docker:1` will always get the latest stable release. thoughts?